app/javascript/config/constants.js

```diff
@@ -297,15 +297,17 @@ export const APPLICATION_NAV = (permissions, userId) => {
       name: "navigation.home",
       to: ROUTES.dashboard,
       icon: "home",
-      validateWithUserPermissions: true
+      validateWithUserPermissions: true,
+      show: true
```

Comment:
Since we are going to skip rendering disabled records maybe we should use another term such as active instead of show. When a record is not active we will skip rendering it.

Reply:
Done!